packages/react-testing/src/element.ts

```diff
@@ -220,18 +222,120 @@ export class Element<Props> implements Node<Props> {
     });
   }
 
+  debug(verbosity?: number, depth?: number): string;
+  debug(options?: DebugOptions): string;
+  debug(options: number | DebugOptions = {}, depth?: number) {
+    return printElement(
+      this,
+      typeof options === 'number' ? {verbosity: options, depth} : options,
+    );
+  }
+
   toString() {
-    const {type} = this;
+    return `<${elementName(this)} />`;
+  }
+}
+
+function printElement(element: Element<any>, options: DebugOptions, level = 0) {
```

Comment:
Is there no existing serializer for a react element we could use?

Reply:
if there is i didn't encounter one

Reply:
looks like are a few options:

- [`react-element-to-jsx-string`](https://github.com/algolia/react-element-to-jsx-string)
- [`pretty-format`](https://github.com/facebook/jest/tree/master/packages/pretty-format) from `jest`
- `React.renderToStaticMarkup(...)`

any of these could work, but really there isn't much code here so i'm not sure how interested i am in pulling in another dependency. the `renderToStaticMarkup` is probably a bit basic, `pretty-format` could be reasonable though if it's already present in the package dependences?

Reply:
i tried out `pretty-format` and it was a no go, we would need to change how we are rendering nodes which isn't really in the scope of this feature. The other options i don't feel make sense to attempt.